Deeploy/TilingExtension/CodeTransformationPasses/SingleBufferingTilingCodeGeneration.py

```
@@ -0,0 +1,150 @@
+class SingleBufferingTilingCodeGeneration(TilingCodeGeneration):
+
+    def __init__(self, externalMemory: str, localMemory: str, dma: AsyncDma):
+        super().__init__(externalMemory, localMemory, dma, 1)
+
+    def _generateTransferScheduleCalls(
+            self, ctxt: NetworkContext, operatorRepresentation: OperatorRepresentation,
+            transferSchedule: List[Dict[str, HyperRectangle]], tensorMemoryConstraintDict: Dict[str,
+                                                                                                TensorMemoryConstraint],
+            tileIdxVar: str, direction: DmaDirection) -> Tuple[NetworkContext, List[CodeSnippet], Set[Future]]:
+        callStack: List[CodeSnippet] = []
+        referenceUpdates: List[CodeSnippet] = []
```

Ruff (0.12.2)

Local variable `referenceUpdates` is assigned to but never used

Remove assignment to unused variable `referenceUpdates`

(F841)